### PR TITLE
adding Overview of Chainguard Enforce

### LINKED ISSUE #99: Add Enforce Overview and Architecture overview to Getting started docs
Feedback from empathy day:

"I think in general we should have an enforce overview and architecture overview, and a little better explanation of what each part of the onboarding steps do"

My thoughts are a section that explains the basic features of enforce: allows you to implement and enforce security policies on your kubernetes fleet, monitor the status of policies in real time, easily update and distribute policies across clusters, require image signatures, SBOMs, and secure registries, etc.

and also a quick overview of the components:
- enforce platform: a SaaS solution that does blah blah
- enforce agent: lightweight agent that runs on your K8s cluster nodes to distribute policies, continuously monitor the security of clusters, and report information to the enforce platform
- chainctl: a command line utility that lets you manage users, groups, define policies, authenticate to chainguard, and check status of clusters at the command line

### PR DESCRIPTION
**Pull request by SharpRake**
## Type of change

This PR adds a directory containing the content and images for an Overview of Chainguard Enforce.

### What should this PR do?

This resolves #99 

### What are the acceptance criteria? 

@amdawson already gave this a review but if he has any further comments or suggestions I'd love to hear them

### How should this PR be tested?

This is a conceptual article, so this will mostly require a copyedit (though be on the lookout for anything that seems wrong or contradictory!)

### CODE REVIEW

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+An organization's software supply chain can contain many possible points of entry for malicious actors and lots of potential for unintended vulnerabilities. Indeed, with recent high-profile security incidents like the SolarWinds breach and the discovery of the Log4Shell vulnerability, [software supply chain security](/software-security/what-is-software-supply-chain-security/) has become a top priority for many organizations.
+
+To help keep organizations' software supply chains secure by default, Chainguard has developed **Chainguard Enforce, a supply chain security solution for containerized workloads**. Chainguard Enforce enables you to ensure continuous compliance by enforcing policies that protect organizations from supply chain threats. Using open source projects and standards that are trusted by the community — like Cosign and Fulcio from the Sigstore project — Chainguard Enforce offers a robust approach to securing your workloads, allowing you to do the following:
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md:20]:
```suggestion
**Chainguard Enforce** offers a supply chain security solution for containerized workloads, which can help keep organizations' software supply chains secure by default. Enabling engineering and security teams to ensure continuous compliance, Chainguard Enforce protects organizations from supply chain threats through policy enforcement. Using open source projects and standards that are trusted by the community — like Cosign and Fulcio from the Sigstore project — Chainguard Enforce offers a robust approach to securing your workloads, allowing the following:
```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+
+## What is Chainguard Enforce?
+
+Chainguard Enforce is a developer platform for end-to-end software supply chain security that operates at the level of Kubernetes fleets. Once you install Enforce onto a Kubernetes cluster and apply a policy, Chainguard Enforce will immediately begin analyzing metadata associated with images running in the cluster to determine whether it satisfies the requirements defined within the policy. Specifically, it inspects the given cluster's [software artifacts](https://console.enforce.dev/policies/catalog), such as an image's software bill of materials (SBOM). Based on the details within the images' metadata, Chainguard Enforce will determine whether those images satisfy the policy.
+
+Chainguard Enforce provides a rich IAM model similar to the likes of AWS and GCP, allowing you to organize policies and clusters into a hierarchy of groups through its Identity and Access Management (IAM) model. When you install Enforce onto a cluster, you associate that cluster with an existing IAM group. Likewise, when you create a policy, you associate it with a group and it is automatically applied to any clusters already associated with that group.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md:35]:
```suggestion
Chainguard Enforce provides a rich [IAM model](../overview-of-enforce-iam-model/) similar to the likes of AWS and GCP, allowing you to organize policies and clusters into a hierarchy of groups through its Identity and Access Management (IAM) model. When you install Enforce onto a cluster, you associate that cluster with an existing IAM group. Likewise, when you create a policy, you associate it with a group and it is automatically applied to any clusters already associated with that group. 
```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+Typically, admission controllers only check policies when Kubernetes resources are being created. If new policies are rolled out, or the state of an application’s compliance degrades over time, then admission controllers will not catch the compliance degradation. Chainguard Enforce, however, continuously scans the cluster against all of the latest policies. This means that if a deployed application violates a new policy or falls out of compliance, it will catch the policy violation before Kubernetes acts on the request. 
+
+Violations can be sent as Cloud Events, so you can respond in the most appropriate way for your policies and applications. This might be to file a JIRA ticket, page the on-call, delete pods, or trigger a CI/CD pipeline to redeploy the application. Regardless of how you configure your response, the violation will be logged for future reference.
+
+> Note: For more information on how continuous verification works, check out our conceptual article on [Understanding Continuous Verification](../understanding-continuous-verification/).
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md:43]: I would remove this note and put the link in line

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+Chainguard Enforce provides a rich IAM model similar to the likes of AWS and GCP, allowing you to organize policies and clusters into a hierarchy of groups through its Identity and Access Management (IAM) model. When you install Enforce onto a cluster, you associate that cluster with an existing IAM group. Likewise, when you create a policy, you associate it with a group and it is automatically applied to any clusters already associated with that group. 
+
+One feature that makes Chainguard Enforce unique is continuous verification. This means that Enforce will scan the cluster on a regular cadence to ensure that it complies with any policies you've applied to it. Chainguard Enforce uses an [admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/) to block any requests that violate a cluster's policy.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md:37]:
```suggestion
One feature that makes Chainguard Enforce unique is [continuous verification](../understanding-continuous-verification/). This means that Enforce will scan the cluster on a regular cadence to ensure that it complies with any policies you've applied to it. Chainguard Enforce uses an [admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/) to block any requests that violate a cluster's policy. 
```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+Violations can be sent as Cloud Events, so you can respond in the most appropriate way for your policies and applications. This might be to file a JIRA ticket, page the on-call, delete pods, or trigger a CI/CD pipeline to redeploy the application. Regardless of how you configure your response, the violation will be logged for future reference.
+
+> Note: For more information on how continuous verification works, check out our conceptual article on [Understanding Continuous Verification](../understanding-continuous-verification/).
+
+The following diagram outlines Chainguard Enforce's various components and how they work together. 
+
+![Diagram outlining Chainguard Enforce's architecture.](enforce-diagram.png)
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md:47]: On the dark background the image is hard to read:

Screenshot 2022-12-15 at 4 48 47 PM

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+`chainctl` uses the familiar `<context> <noun> <verb>` style of CLI interactions. For example, to list groups within the context of Chainguard IAM, you can run `chainctl iam groups list` to receive relevant output.
+
+For more information on how to use `chainctl`, please refer to our [`chainctl` documentation](/chainguard/chainguard-enforce/chainctl-docs/chainctl/).
+
+
+## The Chainguard Enforce Agent
+
+In the context of computing, a software agent is a program that performs certain actions on behalf of a user. A software agent has the authority to make decisions on its own and doesn't necessarily require user input. Accordingly, the Chainguard Enforce Agent is able to analyze a given cluster and enforce any policies you've applied to the cluster without your oversight.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md:65]:
```suggestion
In the context of computing, a software agent is a program that performs certain actions on behalf of a user. A software agent has the authority to make decisions on its own and doesn't necessarily require user input. Accordingly, the Chainguard Enforce Agent is able to analyze a given cluster and enforce any policies you've applied to the cluster without manual human oversight. 
```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+
+## Chainguard Enforce for Git
+
+In addition to ensuring that Kubernetes clusters comply with established policies, Chainguard Enforce also supports Git signature verification for GitHub repositories. Chainguard Enforce for Git is built around [Gitsign](https://docs.sigstore.dev/gitsign/overview/) a component of the [Sigstore project](https://www.sigstore.dev/). Gitsign implements keyless Sigstore to sign Git commits with a valid OpenID Connect identity, like a GitHub or Google account. This allows users to sign their Git commits without the need for GPG keys or a complicated configuration.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md:98]:
```suggestion
In addition to ensuring that Kubernetes clusters comply with established policies, Chainguard Enforce also supports Git signature verification for GitHub repositories. Chainguard Enforce for Git is built around [Gitsign](https://docs.sigstore.dev/gitsign/overview/) a tool that is part of the [Sigstore project](https://www.sigstore.dev/). Gitsign implements keyless Sigstore to sign Git commits with a valid OpenID Connect identity, like a GitHub or Google account. This allows users to sign their Git commits without the need for GPG keys or a complicated configuration.
```

Comment (ltagliaferri) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md:98]: This is a nit, just that it is a tool on its own as part of the Sigstore "suite"

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+## Managing Chainguard Enforce with chainctl
+
+`chainctl` — short for "Chainguard Control" — is the command line interface tool for interacting with Chainguard Enforce. It communicates with the Enforce API, allowing you to manage policies, IAM groups, and cluster connections. Assuming you've been granted access to the product, you can [install `chainctl` for Chainguard Enforce](/chainguard/chainguard-enforce/how-to-install-chainctl/) on any machine you use to manage your Kubernetes clusters.
+
+`chainctl` uses the familiar `<context> <noun> <verb>` style of CLI interactions. For example, to list groups within the context of Chainguard IAM, you can run `chainctl iam groups list` to receive relevant output.
+
+For more information on how to use `chainctl`, please refer to our [`chainctl` documentation](/chainguard/chainguard-enforce/chainctl-docs/chainctl/).
+
+
+## The Chainguard Enforce Agent
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md:63]: Do you think a page on the Enforce Agent that is a little more detailed is warranted?

**Review by ltagliaferri — APPROVED** [review]
This is great, some comments / feedback throughout. I think the tone at the top can be a little more objective and I tried editing in that direction.

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md (hunk 8 above)
Comment (SharpRake) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md:63]: warranted, yes. But probably not a necessity right now

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md (hunk 7 above)
Comment (SharpRake) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md:98]: that's very helpful, I was having a hard time describing it!

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md (hunk 5 above)
Comment (SharpRake) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md:47]: I added a white background and bumped up the font size across the board.

**Pull request merged**